components/gif_picker/components/SearchBar/index.jsx

```diff
@@ -214,10 +201,11 @@ export class SearchBar extends Component {
                         className='search-input-bg'
                         style={style.inputBackground}
                     />
-                    {placeholder}
                     <input
                         className='search-input'
                         name='searchText'
+                        autoFocus={true}
+                        placeholder={'Search Gfycat'}
```

Comment:
This text should be translated, but I'll make a note to do that later since it's not a new thing